### ISSUE TITLE
Consistent required classes

### ISSUE THREAD

**Issue by stijnelskens** [verbb/formie#514]
**Description**
If you make a common field required the parent div gets the class `fui-field-required` but if you add an advanced address field or name field it doesn't get that class.
Can every form get consistent the required `fui-field-required` class so I can work with it?

**Steps to reproduce**
1. Add a common field and make it required
2. Add an address field and make the fields required
3. Check on the frontend the classes on the parent element (above the container)

### Form settings
- Multi-page form:  No
- Submission Method: Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.4
- Craft version: 3.6.15
- Multi-site: No

**Comment by engram-design** [comment]
The tricky part with sub-fields (which are address and multi-name enabled name fields) is that you require individual sub-fields, rather than the entire fields being enabled, which would be the reason for their discrepancy.

So in your scenario, if the `city` field in an address field was marked as required, but no other sub-fields were, you'd still want a `fui-field-required` added to the `fui-field` container?

**Comment by stijnelskens** [comment]
Yes the fui-field container wraps only around 1 input right?
Or am I wrong?
Here is how I see the HTML:
```
<div class="fui-row">
  <div class="fui-field {required comes here}">
    <div class="fui-input-container">
      city input here
    </div>
  </div>
  <div class="fui-field">
    <div class="fui-input-container">
      zip input here
    </div>
  </div>
</div>

```

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.9](https://github.com/verbb/formie/releases/tag/1.4.9)

**Comment by stijnelskens** [comment]
Thanks!
